testing/test-helper.js

```diff
@@ -143,8 +143,7 @@ export function assertScreenReaderElement(element, {index = 0} = {}) {
   expect(computedStyle.getPropertyValue('visibility')).to.equal('visible');
 }
 
-// Use a browserId to avoid cross-browser race conditions
-// when testing in Saucelabs.
+// TODO(amphtml): Remove browserId now that we don't test on Sauce Labs.
```

Comment:
Can you leave the original comment and then add the TODO so we know why browserId was added? Because it's possible we need to keep this for future cross-browser tests.

Reply:
Agreed. Done.